packages/auto-label/test/auto-label.ts

```diff
@@ -275,20 +275,164 @@ describe('auto-label', () => {
     });
   });
 
-  it('responds to backfill label event, backfilling issues with labels', async () => {
-    const payload = require(resolve(fixturesPath, './events/issue-labeled'));
-    const ghRequests = nock('https://api.github.com')
-      .get('/repos/testOwner/testRepo/issues')
-      .reply(200, [
-        {
+  describe('schedule repository', () => {
+    it('responds to a scheduled event', async () => {
+      const ghRequests = nock('https://api.github.com')
+        .log(console.log)
+        .get('/repos/testOwner/testRepo/issues')
+        .reply(200, {
           number: 1,
+        })
+        .get('/repos/testOwner/testRepo/issues/1/labels')
+        .reply(200)
+        .get('/repos/testOwner/testRepo/labels/myGitHubLabel')
+        .reply(200)
+        .post('/repos/testOwner/testRepo/labels', {
+          name: 'myGitHubLabel',
+          color: 'FEFEFA',
+        })
+        .reply(201, [
+          {
+            id: 1811802233,
+            node_id: 'MDU6TGFiZWwxODExODAyMjMz',
+            url:
+              'https://api.github.com/repos/sofisl/mergeOnGreenTest/labels/anotherLabel',
+            name: 'myGitHubLabel',
+            color: 'FEFEFA',
+            default: false,
+            description: null,
+          },
+        ])
```

Comment:
Do we need full replies? Not sure what the best practice is.

Reply:
Good call! Shortened them.